capsule/src/CapsuleSigner.ts

```diff
@@ -306,6 +426,13 @@ export abstract class CapsuleBaseSigner {
     }
   }
 
+  /**
+   * Get the walletId from the Capsule server using the account address.
+   * @param userId UserId registered with the Capsule Server
+   * @param address Address of the account.
+   * @returns The UserId.
+   * @category Private
+   */
   private async getWallet(userId: string, address: string): Promise<any> {
```

Comment:
is there a way we can avoid documenting private methods?

Reply:
Yeah, we can hide private methods  using the `--excludePrivate` flag and replace `@deprecated` with `@hidden`